audio/ID3/Portfile

```diff
@@ -25,10 +25,6 @@ distname            ID3.framework.${version}
 checksums           rmd160  3b41296cca04d641c1e315fe7a93a0443ebbb8a5 \
                     sha256  7d294b2348fc36ac8b068bab15e1b58602aafec098f670cccc52b7436b918270
 
-post-extract {
-    move "${workpath}/id3 tag framework" ${worksrcpath}
-}
-
```

Comment:
Committing this to ports now, without the changes from macports/macports-base#55, will cause the port to fail (`can't find file to patch at input line 3`). And of course even after the changes in macports/macports-base#55 are committed, users will not be running that code until after we release a new version of MacPorts and users have updated to it. So whatever changes are committed to ports for this issue (for all ports, not just this one) need to be made such that they work both with and without the changes from macports/macports-base#55. Compatibility code can be removed from Portfiles after all users have upgraded to the new version of MacPorts.